src/TiciTacaToeyGameEngine.ts

```diff
@@ -428,6 +443,7 @@ class TiciTacaToeyGameEngine implements GameEngine {
             JSON.stringify({ ...response, type: MessageTypes.SPECTATE_GAME })
           );
         });
+        // notify the client of the changes in timings
```

Comment:
Hey do we need to do anything extra for this I think game object should already have this?